Documentation/sourcebuild-in-repos/update-dependencies.md

```diff
@@ -0,0 +1,28 @@
+# Updating a dependency
+
+In most cases, dependency flow should automatically update the packages you depend on.  There are some special considerations and things to keep in mind when doing manual updates.
+
+## External packages
+
+Updating a non-Microsoft (e.g. Newtonsoft.Json 9.0.1 to 13.0.1) or non-Arcade (e.g. ApplicationInsights) package is typically equivalent to adding the new version all over again.  Please contact a [source-build team member](https://github.com/orgs/dotnet/teams/source-build-internal) to discuss this.
```

Comment:
I think it would be better to have the reader log an issue here as a way to start the conversation.  We should also link to the source-build-externals repo as a way to check if the desired version is already included in SB

Reply:
Changed as suggested.